bloco_4/dia_3/exercicios-do-dia/scripts.js

```diff
@@ -0,0 +1,120 @@
+// 1- Para o primeiro exercício de hoje, faça um programa que, dado um valor n qualquer, seja n > 1 , imprima na tela um quadrado feito de asteriscos de lado de tamanho n . Por exemplo:
+// n = 5
+// *****
+// *****
+// *****
+// *****
+// *****
+let n = 5;
+let square = '';
+
+for (let index = 0; index < n; index += 1){
+  square = square+'*';
```

Comment:
Quando vc quer concatenar uma variavel com ele mesma vc pode usar o `+=` ao inves de escrever o nome da variavel novamente. Vou deixar apenas este comentario para nao poluir muito, mas vale para os outros exercicios.
```suggestion
  square += '*';
```